### PR TITLE
Add DownloadSubtitles intent

### PR DESCRIPTION
**Pull request by rinzeb**
Added an intent for downloading subtitles. When playing a show or movie, this will open the "Download subtitles" window, where you can download a subtitle from one of your enabled subtitle addons. I prefer to enable the Kodi setting "Automatically download the first subtitle" in order to download subtitles with just 1 voice command. Alternatively, this requires a "Select" command to download a subtitle.
(Related to issue #49 )

### CODE REVIEW

**Comment by rinzeb** [comment]
Requires a PR on kodi-voice too: https://github.com/m0ngr31/kodi-voice/pull/16

File: speech_assets/SampleUtterances.german.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# A list of utterance variations written in a shorthand syntax
+# Syntax and information
+# Tool to expand the single line into the usable list of utterances
+# http://www.makermusings.com/amazon-echo-utterance-expander/
+# http://www.makermusings.com/two thousand fifteen/seven/twenty one/defining-utterances-for-the-alexa-skills-kit/
```
Comment (jingai) [inline speech_assets/SampleUtterances.german.txt:5]: Why did you add this comment here?  This isn't the file containing the shorthand versions.

**Comment by jingai** [comment]
Thanks for the PR.

I don't think we should use "search for" here though, since that's how we invoke the Global Search addon.

**Comment by jingai** [comment]
Should this return `question()`?  If the auto-download-first-available option is enabled, it doesn't pop a dialog, but there's not really any way for the skill to know that.

**Comment by jingai** [comment]
Merged with the above changes.  Thanks again!
